website/docs/react-native/santoku/design/demo/push-notification/overview.mdx

```diff
@@ -0,0 +1,96 @@
+# プッシュ通知
+
+## 概要
+
+プッシュ通知の機能を検証するデモページです。
+
+- プッシュ通知の許可状態取得
+- プッシュ通知受信の許可ダイアログ表示
+- プッシュ通知受信の仮許可
+- プッシュ通知の送信
+- プッシュ通知の設定画面への遷移
```

Comment:
Please create tasks to add section introducing the notification center ui, including collupsed(Android), summarized(iOS), time-sensitive(iOS).

Reply:
Added!
https://dev.azure.com/ws-4020/develop-mobile-app-casually/_backlogs/backlog/Mobile%20Dev%20Team/Epics/?workitem=16375